components/contribution-flow/index.js

```diff
@@ -798,6 +815,22 @@ class CreateOrderPage extends React.Component {
     return get(this.state, 'stepProfile.settings.hideCreditCardPostalCode', false);
   }
 
+  createFeesOnTopOptions = amount => {
+    const map = new Map([
+      ['10%', 0.1],
+      ['15%', 0.15],
+      ['20%', 0.2],
+    ]);
+    const percentArray = Array.from(map, x => ({
+      label: `${formatCurrency(x[1] * amount, this.getCurrency())} (${x[0]})`,
+      value: x[1] * amount,
+    }));
+    const otherOption = { label: 'I do not want to contribute', value: 0 };
+    const noContributionOption = { label: 'Other', value: 0 };
```

Comment:
Any way we can internationalize these labels?

Reply:
100% - good catch